Gemfile

```diff
@@ -19,6 +19,7 @@ end
 
 gem 'acts_as_tenant'
 gem 'jbuilder', '~> 2.0'
+gem "manageiq-password", :git => "https://github.com/ManageIQ/manageiq-password", :branch => "master"
```

Comment:
The git based gems are listed below (see topological_inventory-api-client line)

Reply:
Why is this pointing to git when we have a released version on rubygems.org? https://rubygems.org/gems/manageiq-password

Reply:
@bdunne - I used what Topology is using - https://github.com/ManageIQ/topological_inventory-api/blob/master/Gemfile#L11  - I wasn't aware the gem lived in rubygems.  I can make that change, but is there any special behavior in the `git` version vs the released version that may be important?